parent/pom.xml

```
@@ -18,7 +18,7 @@
   </description>
 
   <properties>
-    <version.quarkus>3.2.6.Final</version.quarkus>
+    <version.quarkus>3.7.1</version.quarkus>
```

🔧 Shall we try out the latest patch?
❓ What will be the benefit? They did have breaking changes in a patch before so we can test it just in case before they release 3.8.
```suggestion
    <version.quarkus>3.7.2</version.quarkus>
```